README.md

```diff
@@ -22,29 +22,16 @@ Command to install ConfiGator
 pipx install configator
 ```
 
-### Commands to Change Specific Information Within a File
+### Command to Change Specific Configurations
 
-Command to change name
+The command below will show a list of every available command to change
+specific configurations in `config/gatorgrader.yml`, or `build.gradle`
+within a file of your choice.
```

Comment:
```suggestion
specific configurations in `config/gatorgrader.yml` or `build.gradle`.
```

Reply:
`within a file of your choice` is redundant here; you already list the files.

Reply:
By the way, you can easily accept this suggestion with the **Commit suggestion** button above.